packages/flutter_tools/lib/src/ios/xcresult.dart

```diff
@@ -263,36 +272,27 @@ enum XCResultIssueType {
 
 // A typical location url string looks like file:///foo.swift#CharacterRangeLen=0&EndingColumnNumber=82&EndingLineNumber=7&StartingColumnNumber=82&StartingLineNumber=7.
 //
-// This function converts it to something like: file:///foo.swift:<StartingLineNumber>:<StartingColumnNumber>.
-String _convertUrlToLocationString(String url){
-  if (url.isEmpty) {
-    return '';
+// This function converts it to something like: /foo.swift:<StartingLineNumber>:<StartingColumnNumber>.
+String? _convertUrlToLocationString(String url) {
+  final Uri? fragmentLocation = Uri.tryParse(url);
+  // Maybe log something here
```

Comment:
This was a note for you, not intended to be checked in... 🙂

Reply:
oh, the copy-paste :p